arches_for_science/templates/views/components/plugins/importer-configuration.htm

```diff
@@ -3,6 +3,7 @@
     <h4><span>{% trans "Importer Configurations" %}</span></h4>
     <span>{% trans "Define how to import your x/y data" %}</span>
     <div>
+        <div class="afs-error-message ep-alert-red" style="padding:5px;" data-bind="visible: rendererInUse">{% trans "This importer is in use - it should be replace it in all files where it is used before deleting." %}</div>
```

Comment:
Looks like a typo in the error text. Maybe 'replace it' should just be 'replaced'